packages/shared/src/components/search/SearchReferralBanner.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ReactElement } from 'react';
+import { Button } from '../buttons/Button';
+import { KeyReferralIcon } from '../icons';
+import { IconSize } from '../Icon';
+import { ReferralCampaignKey, useFeatureCampaign } from '../../hooks';
+
+export function SearchReferralBanner(): ReactElement {
+  const { referralCampaign, canInvite } = useFeatureCampaign({
+    campaignKey: ReferralCampaignKey.Search,
+  });
+
+  if (!canInvite) {
+    return null;
+  }
+
+  const { availableCount, noKeysAvailable } = referralCampaign;
+
+  return (
+    <div className="flex flex-col gap-4 p-4 w-full rounded-14 bg-theme-overlay-from">
```

Comment:
The background feels incomplete here?

Reply:
Do you want me to check with design?

![Screenshot 2023-10-18 at 6 30 26 PM](https://github.com/dailydotdev/apps/assets/13744167/b85fbc53-e146-429c-9ba2-221a7d45733a)

![Screenshot 2023-10-18 at 6 30 38 PM](https://github.com/dailydotdev/apps/assets/13744167/e8ecdb24-b02b-4bfb-b1c8-92c10ebaa61d)